packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/index.scss

```diff
@@ -277,6 +277,7 @@
 
             & .dc-text {
                 width: 10rem;
+                cursor: pointer;
```

Comment:
can we put `cursor: pointer` in pseudo selector `:hover` ? @shafin-deriv

Reply:
Updated